libsolidity/codegen/ir/IRLValue.cpp

```diff
@@ -110,3 +116,7 @@ string IRStorageItem::storeValue(string const& _value, Type const& _sourceType)
 	}
 }
 
+string IRStorageItem::setToZero() const
+{
+	solUnimplemented("Delete for storage location not yet implemented");
```

Comment:
I think the current implementation should work fine for value types, but we can also keep it like that.